grails-app/views/SDS/index.gsp

```diff
@@ -10,15 +10,15 @@
 
 <body>
 <h2>Sensitive Data Service</h2>
-The Sensitive Data Service (SDS) is the ALA mechanism for providing security over data sensitivities. At the moment it supports:
+The Sensitive Data Service (SDS) is the mechanism for providing security over data sensitivities. At the moment it supports:
 <ul>
     <li>Conservation Sensitivity, whereby sensitive species are denatured according to state rules. </li>
     <li>Pest Sensitivity. Species are passed through a set of rules based on categories as defined in the <b>Plant Biosecurity Sensitive Data Service</b></li>
 </ul>
-For more information see <a href="http://www.ala.org.au/faq/data-sensitivity/">Data Sensitivity</a>.
+For more information see <a href="${grailsApplication.config.sdsFAQ.url}">Data Sensitivity</a>.
 <h2>Testing</h2>
 <p>
-The Sensitive Data Service can be tested through the <a href="http://sandbox.ala.org.au">sandbox</a>. Simply upload your list of species and coordinates and the
+The Sensitive Data Service can be tested through the <a href="${grailsApplication.config.sandbox.url}">sandbox</a>. Simply upload your list of species and coordinates and the
```

Comment:
default sandbox.url seems to be missing from application.yml

Reply:
Added in 5b4ffac97b318e17cc792f018654c097bde50fb8